src/components/table/table.stories.mdx

```diff
@@ -1,6 +1,26 @@
 import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
 import basicDemo from './demo/basic.twig';
 import data from './demo/data.json';
+const basicTableCodeExample = (tableClass) => {
+  let withArgs = '';
+  if (tableClass) {
+    withArgs = ` with { table_class: '${tableClass}' }`;
+  }
```

Comment:
Nit: would it be clearer/more concise with a ternary? If you don't think so that's totally fine, I'm not totally sure which I like better.